lib/vehicle.rb

```diff
@@ -5,14 +5,22 @@ class Vehicle
               :year,
               :make,
               :model,
-              :engine
+              :engine,
+              :registration_date
 
+  vehicle_details = {:vin => "vin",
+  :year => :year,
+  :make => :make,
+  :model => :model,
+  :engine => :engine}
```

Comment:
In Ruby, it's very rare to define a variable outside of a method like this. `vehicle_details` should return a hash which is populated with the attributes that you set up in the initialize method. Wrapped in a method, that might look like
```
def vehicle_details
  {
    vin: @vin,
    make: @make,
    model: @model,
    engine: @engine
  }
end
```